src/main/java/Flash.java

```diff
@@ -0,0 +1,138 @@
+import java.io.File;
+import java.sql.Array;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+public class Flash {
+
+    private static final List<Task> tasks = new ArrayList<>();
+
+    public static void displayTasks() {
+        System.out.println("____________________________________________________________");
+        for(int i = 0; i < tasks.size(); i++) {
+            System.out.println(i+1 + "." + tasks.get(i));
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void markTask(String input) throws FlashException {
+        try {
+            int taskNumber = Integer.parseInt(input.split(" ")[1]) - 1;
+            Task task = tasks.get(taskNumber);
+            task.markDone();
+            System.out.println("____________________________________________________________");
+            System.out.println("Nice! I've marked this task as done:");
+            System.out.println(" " + task);
+            System.out.println("____________________________________________________________");
+        } catch (Exception e) {
```

Comment:
Add a specific check for missing task numbers to improve the error handling instead of generic exception.